ci-operator/step-registry/openshift-qe/ingress-perf/openshift-qe-ingress-perf-commands.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+oc config view
+oc projects
+python --version
+pushd /tmp
+python -m virtualenv ./venv_qe
+source ./venv_qe/bin/activate
+
+ES_PASSWORD=$(cat "/secret/password")
+ES_USERNAME=$(cat "/secret/username")
+
+# Clone the e2e repo
+git clone https://github.com/cloud-bulldozer/e2e-benchmarking
+pushd e2e-benchmarking/workloads/ingress-perf
+
+# ES Configuration
+export ES_SERVER="https://$ES_USERNAME:$ES_PASSWORD@search-ocp-qe-perf-scale-test-elk-hcm7wtsqpxy7xogbu72bor4uve.us-east-1.es.amazonaws.com"
+export ES_INDEX='router-test-results'
+
+# Start the Workload
+CONFIG=config/aws-standard.yml ./run.sh ITERATIONS=5
```

Comment:
can we add the config parameter as an env variable with aws-standard.yml as the default in the ref file

Reply:
Shall we also make ITERATIONS as env variable?